src/interface/blas2/spr2.cpp.in

```diff
@@ -0,0 +1,43 @@
+/***************************************************************************
+ *
+ *  @license
+ *  Copyright (C) Codeplay Software Limited
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  For your convenience, a copy of the License has been included in this
+ *  repository.
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *  SYCL-BLAS: BLAS implementation using SYCL
+ *
+ *  @filename spr2.cpp.in
+ *
+ **************************************************************************/
+#include "container/sycl_iterator.hpp"
```

Comment:
Could you double check if these headers are required here or not?

Reply:
Hi @muhammad-tanvir-1211 thank you for noticing it. I checked and it seems we do not need the following headers:
```
#include "container/sycl_iterator.hpp"
#include "operations/blas2_trees.hpp"
#include "views/view_sycl.hpp"
```
I will remove them. 
All interfaces introduce these types of headers, should we remove them?